compute/admin_guide/install/install_defender/install_app_embedded_defender_fargate.adoc

```diff
@@ -183,51 +183,120 @@ Prisma Cloud supports the following task definition formats:
 }
 ----
 
-*Example of the equivalent task definition as a CloudFormation template:*
+*Example of the equivalent task definition as a CloudFormation template in JSON:*
 
 ----
 {
-  "Type" : "AWS::ECS::TaskDefinition",
```

Comment:
@ssugandh I don't think this example should have been removed, it is still valid.
As I explained we now support two options:
1. providing just the task definition (same as it was before)
2. providing a full CloudFormation template (new)

So I think we should keep the existing example as the equivalent task in CloudFormation the Native Fargate task, and after it, provide another example of a full CloudFormation template in JSON, and then one example in YAML too (either just the task definition or a full CloudFormation template)

Reply:
I added back the removed example.